tests/conftest.py

```diff
@@ -1,9 +1,14 @@
+from typing import TYPE_CHECKING
+
 import pytest
 from hexbytes import HexBytes
 
 from eip712.common import create_permit_def
 from eip712.messages import EIP712Message, EIP712Type
 
+if TYPE_CHECKING:
+    from eth_pydantic_types.abi import address, bytes32, string, uint256
```

Comment:
Not performance critical, so might be worth doing it like this
```suggestion
from eth_pydantic_types import abi
```

And then below do like :

```py
...
inner: abi.uint256
```